monitoring/mongodb/alerts.yaml

```diff
@@ -139,3 +139,24 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      sum by (pod) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"} >= 1)
```

Comment:
We can have all these states (excluding arbitrer)
https://www.mongodb.com/docs/manual/reference/replica-states/

I wonder if we should have alert on all the unwanted states, and not only restrict the alert on the `RECOVERING` state? Like, detecting the state is neither 0; 1 nor 2. We may also need some specific alert depending on the state:

* For 3 (RECOVERING)
  * MongoDB secondaries will periodically enter this state during high load, if they fail to keep up with the incoming data. In general, this state is not harmful.
  * But the issue arises when it stays for too long (the op log is overwritten), that's why I think the `for: 5m` should be changed to something higher, like 24h...
  * When we perform the "full init sync" procedure, the affected mongodb instance will also end up in this state, and depending on the cluster' size, it can take hours, but this is expected, in this case...
  * It's great to have a separate alert, so we can ask the reader to check the logs, and if the instance cannot recover, it will have a specific log (we document it in the troubleshooting procedure - but not sure we want to reference it from Zenko).

* For 5 (STARTUP2) is also fine, but again, if it stays for too long (> several hours), the instance may be stuck.
* For 6 (UNKNOWN), 8 (DOWN) and 10 (REMOVED), these are unwanted states, so we can alert directly without waiting.
* For 9 (ROLLBACK) this is fine as well.

Reply:
done here :https://github.com/scality/Zenko/pull/2142/commits/6bc518d4047053558ea27dc5a23907ad35493474#